docs/pages/layouts.js

```diff
@@ -17,105 +17,80 @@ card(
 
 card(
   <Example
-    description="Responsive &amp; RTL friendly form layout."
+    description="Responsive &amp; RTL-friendly form layout."
     name="Form Layout"
     defaultCode={`
-<Box
-  display="flex"
-  marginStart={-3}
-  marginEnd={-3}
-  marginBottom={-3}
-  marginTop={-3}
-  wrap
-  width="100%"
+<Flex
   direction="column"
+  gap={6}
   maxWidth={800}
+  width="100%"
+  wrap
 >
-  <Box flex="grow" paddingX={3} paddingY={3}>
-    <Heading size="sm" accessibilityLevel={2}>
-      Form Title
-    </Heading>
-  </Box>
+  <Heading size="sm" accessibilityLevel={2}>
+    Form Title
+  </Heading>
 
-  <Box flex="grow" paddingX={3} paddingY={3}>
-    <TextField
-      label="TextField 1"
-      id="textfield1"
-      onChange={() => {}}
-      placeholder="Placeholder"
-    />
-  </Box>
+  <TextField
+    label="TextField 1"
+    id="textfield1"
+    onChange={() => {}}
+    placeholder="Placeholder"
+  />
 
-  <Box flex="grow" paddingX={3} paddingY={3}>
-    <Box
-      display="flex"
-      wrap
-      marginStart={-3}
-      marginEnd={-3}
-      marginBottom={-3}
-      marginTop={-3}
-    >
-      <Box flex="grow" paddingX={3} paddingY={3}>
-        <TextField
-          label="TextField 2"
-          id="textfield2"
-          onChange={() => {}}
-          placeholder="Placeholder"
-        />
-      </Box>
-      <Box flex="grow" paddingX={3} paddingY={3}>
-        <TextField
-          label="TextField 3"
-          id="textfield3"
-          onChange={() => {}}
-          placeholder="Placeholder"
-        />
-      </Box>
+  <Box
```

Comment:
I think the elimination of this outer box has changed the responsive behavior:

With Box:

![Screen Shot 2021-09-09 at 8 50 12 PM](https://user-images.githubusercontent.com/5125094/132796944-7f3254de-3bd7-4ed2-aff8-1a8e92b8fdd1.png)

New Version:

![Screen Shot 2021-09-09 at 8 50 19 PM](https://user-images.githubusercontent.com/5125094/132796946-c7956dce-1865-4bfe-8f57-d7faabf87e90.png)

Reply:
That was intentional, as I was seeing the stacking (wrapping) behavior in the prod example. Do you think the two adjacent fields shouldn't stack at a certain width? I added the minWidth to prevent the fields from getting too small. Totally open to changing this though